src/index.html

```diff
@@ -24,8 +24,78 @@
         <link rel="preconnect" href="https://connect.facebook.net" crossorigin="anonymous">
         <link rel="prefetch" href="<%= process.env.API_ORIGIN %>/apps/cms/api/books/?format=json">
         <link rel="prefetch" href="<%= process.env.API_ORIGIN %>/apps/cms/api/footer/?format=json">
-        <link rel="stylesheet" href="https://ram.openstax.org/osano/osano.css">
-        <script src="https://cmp.osano.com/AzZqbXTbzhHsU3cv1/68d8e8ae-4024-4489-a000-72766ad284a6/osano.js"></script>
+        <script>
```

Comment:
Is it possible to move this to a file and have it serve properly?

Reply:
Since the goal is to move it to GTM soon and remove it from this repo, I think it's not worth moving it? We might not even release it like this.